src/Contracts/Syncable.php

```diff
@@ -15,4 +15,6 @@ public function getCentralModelName(): string;
     public function getSyncedAttributeNames(): array;
 
     public function triggerSyncEvent(): void;
+
+    public function getResourceCreationAttributes(): array|null; // todo come up with a better name
```

Comment:
Maybe `getCreationAttributeNames` is better and compatible with `getSyncedAttributeNames`

Reply:
What do you mean? The methods work differently, they return values in different format.

Reply:
Sorry, i haven't seen the  possibility to return attribute values instead of attribute names.
But i think the name of method could be `getCreationAttributes`

Reply:
Ah, yeah, if we're not using the "resource" terminology anywhere in the contract then that'd probably work. @abrardev99 can you make the change?

My only concern with too generic names like that is that they could conflict with Laravel code eventually or *feel* like they're something Laravel-related. When it should be clear that they're related to Tenancy resource syncing.

Reply:
> @abrardev99 can you make the change?

Sure I can.

Reply:
> My only concern with too generic names like that is that they could conflict with Laravel code eventually or feel like they're something Laravel-related. When it should be clear that they're related to Tenancy resource syncing.

After @stancl comment, 
unfortunately, i think that method name could be `getSyncedCreationAttributes`
is compatible with `getSyncedAttributeNames`

Reply:
I like that one, BTW.

Reply:
@stancl Renamed the method and added todo to come up a better name